apps/dashboard/src/utils/s3-upload-file.ts

```diff
@@ -0,0 +1,33 @@
+// Expected response: 204 No Content. Returns resource URL if successful.
```

Comment:
Maybe handle when it's not 204?

Reply:
It throws an error if a 204 is not returned. You mean the frontend code that it the top caller? Like i.e.

```ts      try {
        fileUrl = await handleUpload(data.file)
      } catch (e) {
        notification.fail({
          title: "Feil",
          message: "Kunne ikke laste opp fil. ",
        })
      }
      
      
      ```
      
      in `apps/dashboard/src/modules/offline/modals/create-offline-modal.tsx` instead of just calling handleUpload()

Reply:
I created an issue on exploring monitoring services, https://linear.app/dotkom/issue/DOT-432/explore-monitoring-services